uaclient/entitlements/tests/test_cc.py

```diff
@@ -64,16 +64,23 @@ class TestCommonCriteriaEntitlementUserFacingStatus:
     @mock.patch(M_REPOPATH + "os.getuid", return_value=0)
     @mock.patch("uaclient.util.get_platform_info")
     def test_inapplicable_on_invalid_affordances(
-        self, m_platform_info, m_getuid, arch, series, version, details, tmpdir
+        self,
+        m_platform_info,
+        m_getuid,
+        arch,
+        series,
+        version,
+        details,
+        FakeConfig,
     ):
         """Test invalid affordances result in inapplicable status."""
         unsupported_info = copy.deepcopy(dict(PLATFORM_INFO_SUPPORTED))
         unsupported_info["arch"] = arch
         unsupported_info["series"] = series
         unsupported_info["version"] = version
         m_platform_info.return_value = unsupported_info
-        cfg = config.UAConfig(cfg={"data_dir": tmpdir.strpath})
-        cfg.write_cache("machine-token", CC_MACHINE_TOKEN)
+        cfg = FakeConfig()
```

Comment:
I think we can use `FakeConfig.for_attached_machine(machine_token=CC_MACHINE_TOKEN)` instead